app/presenters/two_factor_authentication/sms_selection_presenter.rb

```diff
@@ -4,6 +4,15 @@ def method
       :sms
     end
 
+    def info
+      if configuration.present?
+        t(
+          'two_factor_authentication.login_options.sms_info_html',
```

Comment:
I was sorta confused why this string already existed. I see that it exists here, despite the fact that (a) it'd never be reached and (b) if it were reached, it would produce the wrong string, since we're not interpolating the `%{phone}` variable.

https://github.com/18F/identity-idp/blob/66b2423223777b8c88b536b44ef299bd83e24bd5/app/presenters/two_factor_authentication/selection_presenter.rb#L53-L56

Should we remove those cases from the default class?